test/searchMPS.jl

```diff
@@ -0,0 +1,94 @@
+using MetaGraphs
+using LightGraphs
+using GraphPlot
+
+L = 4
+N = L^2
+
+instance = "$(@__DIR__)/instances/$(N)_001.txt"  
+
+ig = ising_graph(instance, N)
+set_prop!(ig, :β, 1.) #rand(Float64))
+r = fill(2, N)
+set_prop!(ig, :rank, r)
+set_prop!(ig, :dβ, 0.01)
+
+ϵ = 1E-8
+D = 16
+var_ϵ = 1E-8
+sweeps = 30
+type1 = "log"
+type2 = "lin"
+β = [get_prop(ig, :β)]
+dβ = [get_prop(ig, :dβ)]
+control1 = MPSControl(D, var_ϵ, sweeps, β, dβ, type1) 
+control2 = MPSControl(D, var_ϵ, sweeps, β, dβ, type2) 
+
+states = all_states(get_prop(ig, :rank))
+ϱ = gibbs_tensor(ig)
+@test sum(ϱ) ≈ 1
```

Comment:
This is the only test in this file and it is trivial. Please ad some more tests.